lib/datadog/profiling.rb

```diff
@@ -82,6 +82,18 @@ def self.configuration
       )
     end
 
+    # Starts the profiler, if the profiler is supported by in
+    # this runtime environment and if the profiler has been enabled
+    # in configuration.
+    #
+    # @return [Boolean] `true` if the profiler has successfully started, otherwise `false`.
+    # @public_api
+    def self.start_if_enabled
+      # Getting the profiler instance triggers start as a side-effect;
+      # otherwise we get nil
+      !!Datadog.send(:components).profiler
+    end
```

Comment:
YAY FOR INTEGRATION TESTS :)

Oops I totally lied to you. 🤦‍♂️  See if you can spot what I totally forgot AND that the tests were catching because the webserver DID fork.

```suggestion
    def self.start_if_enabled
      # If the profiler was not previously touched, getting the profiler instance triggers start as a side-effect
      # otherwise we get nil
      profiler = Datadog.send(:components).profiler
      # ...but we still try to start it BECAUSE if the process forks, the profiler will exist but may
      # not yet have been started in the fork
      profiler.start if profiler

      !!profiler
    end
```